packages/grafana-ui/src/components/GraphNG/nullToFormattedValue.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  // getValueFormat,
+  DataFrame,
+  // ThresholdsMode
+} from '@grafana/data';
+
+export function nullToFormattedValue(frame: DataFrame) {
+  frame.fields.forEach((f) => {
+    const noValue = +f.config?.noValue!;
+    // const disp = getValueFormat(
+    //   f.config.thresholds?.mode === ThresholdsMode.Percentage ? 'percent' : f.config.unit ?? ''
+    // );
+
+    if (!Number.isNaN(noValue)) {
+      const values = f.values.toArray();
+      for (let i = 0; i < values.length; i++) {
+        if (values[i] === null) {
+          values[i] = '< '; // TODO: Update color mapping to be value and not label dependent
```

Comment:
Some attention to this part in particular would be welcome. Currently color mapping is happening based on label (to simplify thresholds?) however this seems like a bit of an awkward way to map colors. Seems like it should be by value?

Reply:
I removed this in favor of a value comparison higher up the chain.